docs/getting-started.md

```diff
@@ -1,36 +1,85 @@
-# Getting started with Cog
+# Quickstart
 
-First step is to start a server. You'll need to point it at a Docker registry to store the Docker images For example, [create one on Google Cloud Container Registry](https://cloud.google.com/container-registry/docs/quickstart).
+> **Note:** This guide is a work in progress! Please excuse the brevity. Hopefully it'll give you an idea of how it works, though.
 
-Then, start the server, pointing at your Docker registry:
+This guide will help you understand how Cog works. We're going to take a model that somebody else has built and trained and run it with Cog.
 
-    cog server --port=8080 --docker-registry=gcr.io/some-project/cog
+You'll need git-lfs installed to get the trained model weights. On Mac, run `brew install git-lfs`.
 
-Next, let's build a model. We have [some models you can play around with](https://github.com/replicate/cog-examples). Clone that repository (you'll need git-lfs) and hook up that directory to your Cog server:
+First, clone our examples repository and cat a look at the `inst-colorization` model:
 
-    cd example-models/inst-colorization/
-    cog model set http://localhost:8080/examples/inst-colorization
+    git clone https://github.com/replicate/cog-examples.git
```

Comment:
I made this repo private, use https://github.com/andreasjansson/InstColorization instead